multicluster-config/workload-values.yaml

```diff
@@ -33,16 +33,30 @@ kube-prometheus-stack:
     enabled: true
   grafana:
     enabled: false
-  prometheus: 
+  prometheus:
     prometheusSpec:
       externalLabels:
         # !IMPORTANT: each cluster must have prometheus external label 'cluster' with unique value.
         # This will be used as cluster discriminator when metrics are aggregated on observer cluster.
         cluster: "workload-cluster"
       replicas: 2
       thanos:
-        objectStorageConfig: 
+        objectStorageConfig:
           name: thanos-objstore-config
           key: objstore.yml
-    thanosService: 
+    thanosService:
       enabled: true
+openshift:
+ # Keep only the service accounts used by workload cluster, e.g. don't create account for Grafana
+  serviceAccounts:
```

Comment:
Why are there the same values as in the `chart/values-openshift.yaml`?

Reply:
Ok, I know why. When `Helm` merge values it will overwrite all in `openshift.serviceAccounts`, right?

Reply:
It does not create accounts for observer-only services (grafana, loki). 

According to this [issue](https://github.com/helm/helm/issues/6569) Helm  overwrites the whole list, because it's immutable.